data/templates/css/wikimedia.scss

```diff
@@ -431,15 +496,48 @@ html {
             clear: right;
             float: right;
             width: $in-cols * 11 !important;
+            margin-left: 20px;
         }
     }
 }
 
-/* Composite resolutions */
+/////
+// Composite Resolutions
+/////
+
+@media (max-width: 720px) {
+    .infobox, 
+    .infobox_v2 {
+        width: 100% !important;
+    }
+}
+
 @media (device-width: 720px) and (device-height: 480px),
        (device-width: 720px) and (device-height: 576px) {
-    .mw-body {
-        font-family: $font1-composite;
-        font-size: 19.4px;  /* 22px * 88% because spec was at 110% scale not 125% */
+    #inside-content {
+        font-family: $font-primary-composite;
+
+        h2,
+        .firstHeading,
+        .eos-article-title {
+            border-bottom: 1px solid $divider-color-composite;
+        }
+
+        .disclaimer {
+            border-top: 1px solid $divider-color-composite;
+        }
+    }
+
+    .infobox, 
+    .infobox_v2,
+    .gallerytext,
+    .wikitable,
+    tbody,
+    .thumbcaption,
+    .disclaimer {
+        font-size: 1.0rem !important;  // 20px // Root: 20px
     }
 }
+
+
+
```

Comment:
I didn't see this on wikihow articles though, so maybe is something related only to the wikipedia scss?